projects/hslayers/ng-package.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "../../node_modules/ng-packagr/ng-package.schema.json",
   "dest": "../../dist/hslayers",
-  "assets": ["src/assets"],
+  "assets": ["src/assets", "src/css/app.scss", "src/css/whhg-font/css/whhg.css"],
```

Comment:
app.scss should probably be hslayers-bootstrap.scss

Reply:
thanks, forgot about ng-package.